include/meta/meta_fwd.hpp

```diff
@@ -35,8 +37,12 @@ namespace meta
 {
     inline namespace v1
     {
+#ifdef __cpp_lib_integer_sequence
+        using std::integer_sequence;
+#else
         template <typename T, T...>
         struct integer_sequence;
+#endif
```

Comment:
libc++ macros are generally `LIBCPP_XXX`; `__cpp_lib_XXX` is the format for SD-6 feature test macros for library features. `__cpp_lib_integer_sequence` came from [N4440](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2015/n4440.html#recs.cpp14).